content/js/query.md

```diff
@@ -136,10 +117,83 @@ the relation with the current user.
 For example, to query for records owned by a user that the current user is following:
 
 ```javascript
-query.havingRelation("_owner", skygear.relation.Following)
+query.havingRelation("_owner", skygear.relation.Following);
+query.notHavingRelation(...);
+```
+
+See [Social](/js/guide/relation) section for more relations.
+
+### Geolocation condition
+
+Please visit [Geo-location](/js/guide/geolocation) section for more.
+
+<a name="pagination-ordering"></a>
+## Pagination and Ordering
+
+### Ordering of records
+
+We can sort the records based on certain field in ascending or descending order.
+
+``` javascript
+query.addAscending('age');    // sorted by age increasing order
+query.addDescending('price'); // sorted by price decreasing order
```

Comment:
- better mention that the order of the function calls imply the order of sorting the fields, e.g. this example translates to the SQL `ORDER BY age, price DESC`
- how about `NULLS FIRST` or `NULLS LAST` in SQL?